<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QFrame" name="contparamsframe">
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>150</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <layout class="QVBoxLayout" name="leftverticalLayout">
           <item>
            <layout class="QHBoxLayout" name="equationsHLayout">
             <item>
              <widget class="QLabel" name="eqn_label">
               <property name="minimumSize">
                <size>
                 <width>50</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>Equation</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="eqn_combobox">
               <property name="minimumSize">
                <size>
                 <width>50</width>
                 <height>0</height>
                </size>
               </property>
               <item>
                <property name="text">
                 <string>test1</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>test2</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="initcond_label">
               <property name="text">
                <string>Initial condition</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="initcond_entry">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>0</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="initcond_pushButton">
               <property name="text">
                <string>load</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="contparamsHLayout">
             <item>
              <widget class="QLabel" name="ds_label">
               <property name="minimumSize">
                <size>
                 <width>30</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>ds</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="ds_entry"/>
             </item>
             <item>
              <widget class="QLabel" name="atol_label">
               <property name="minimumSize">
                <size>
                 <width>30</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>atol</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="atol_entry"/>
             </item>
             <item>
              <widget class="QLabel" name="w_x_label">
               <property name="minimumSize">
                <size>
                 <width>30</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>w_x</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="w_x_entry"/>
             </item>
             <item>
              <widget class="QLabel" name="direction_label">
               <property name="text">
                <string>direction</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="direction_combobox">
               <item>
                <property name="text">
                 <string>forward</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>backward</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="branchHLayout">
             <item>
              <widget class="QLabel" name="xaxis_label">
               <property name="text">
                <string>x axis</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="xaxis_combobox"/>
             </item>
             <item>
              <widget class="QLabel" name="yaxis_label">
               <property name="text">
                <string>y axis</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="yaxis_combobox"/>
             </item>
             <item>
              <widget class="QCheckBox" name="saveprofile_checkbox">
               <property name="text">
                <string>save profile</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="save_branch_pushButton">
               <property name="text">
                <string>save branch data</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="rightverticalLayout">
           <item>
            <layout class="QHBoxLayout" name="initcondHLayout">
             <item>
              <widget class="QLabel" name="branch_label">
               <property name="text">
                <string>Branch name</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="branch_entry"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="contsettingsHLayout">
             <item>
              <widget class="QPushButton" name="playpause_pushButton">
               <property name="text">
                <string>play / pause</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="timeHLayout">
             <item>
              <widget class="QLabel" name="elapsedtime_label">
               <property name="text">
                <string>Elapsed time: 0 s</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="avgtime_label">
               <property name="text">
                <string>Avg time per point: 0s</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="mainplotframe">
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>400</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="MplWidget" name="mainplot" native="true">
           <property name="minimumSize">
            <size>
             <width>400</width>
             <height>400</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QFrame" name="eqnparams_frame">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>100</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <widget class="QLabel" name="label_11">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Equation Parameters</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="params_HLayout">
           <item>
            <widget class="QLabel" name="param1_label">
             <property name="text">
              <string>theta</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="param1_entry"/>
           </item>
           <item>
            <widget class="QLabel" name="param2_label">
             <property name="text">
              <string>epsilon</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="param2_entry"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="contlog_frame">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>100</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QLabel" name="contlogtitle_label">
             <property name="font">
              <font>
               <family>Open Sans</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Contination Log</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="log_label">
             <property name="text">
              <string>Step 0. No Continuation</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="secondaryplot_frame">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>400</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="MplWidget" name="secondaryplot" native="true">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>200</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Branch</string>
    </property>
    <addaction name="actionNew_branch"/>
    <addaction name="actionLoad_branch"/>
    <addaction name="actionSave_branch"/>
    <addaction name="actionDelete_branch"/>
    <addaction name="actionSplit_branches"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuFile_2">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile_2"/>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave_branch">
   <property name="text">
    <string>Save branch</string>
   </property>
  </action>
  <action name="actionLoad_branch">
   <property name="text">
    <string>Load branch</string>
   </property>
  </action>
  <action name="actionNew_branch">
   <property name="text">
    <string>New branch</string>
   </property>
  </action>
  <action name="actionSplit_branches">
   <property name="text">
    <string>Split branches</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionDelete_branch">
   <property name="text">
    <string>Delete branch</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplWidget</class>
   <extends>QWidget</extends>
   <header>mplwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
